src/public/resolvers/item.ts

```diff
@@ -21,7 +22,7 @@ export async function collection(
       return null;
     }
   } catch (err) {
-    console.log(err);
+    serverLogger.error(err);
```

Comment:
not a blocker, but would be nice to have an error message here with context, e.g.

```
serverLogger.error('error loading collection by slug in item resolver', {
    err
});
````

Reply:
ill merge this PR for now but add this in my next one for this repo!